tests/e2e/tests/content-manager/relations/base.spec.ts

```diff
@@ -0,0 +1,163 @@
+import {
+  connectRelation,
+  reorderRelation,
+  verifyRelationsOrder,
+  disconnectRelation,
+} from '../../../utils/relation-utils';
+import { createContent, FieldValue, saveContent } from '../../../utils/content-creation';
+
+const RELATION_SOURCE_URL =
+  /\/admin\/content-manager\/collection-types\/api::relation-source.relation-source\/(?!create)[^/]/;
+const RELATION_TARGET_URL =
+  /\/admin\/content-manager\/collection-types\/api::relation-target.relation-target\/(?!create)[^/]/;
+
+const createRelationSourceFields = (rawFields: {
+  name?: string;
+  oneToOneRel?: string;
+  oneToManyRel?: string[];
+  manyToOneRel?: string;
+  manyToManyRel?: string[];
+  oneWayRel?: string;
+  selfRel?: string[];
+}) => {
+  const { name, oneToOneRel, oneToManyRel, manyToOneRel, manyToManyRel, oneWayRel, selfRel } =
+    rawFields;
+
+  // Return FieldValue[]
+  const fields: FieldValue[] = [];
+
+  if (name) {
+    fields.push({ name: 'name', type: 'text', value: name } satisfies FieldValue);
+  }
+
+  if (oneToOneRel) {
+    fields.push({
+      name: 'oneToOneRel',
+      type: 'relation',
+      value: [{ label: oneToOneRel }],
+    } satisfies FieldValue);
+  }
+
+  if (oneToManyRel) {
+    fields.push({
+      name: 'oneToManyRel',
+      type: 'relation',
+      value: oneToManyRel.map((label) => ({ label })),
+    } satisfies FieldValue);
+  }
+
+  if (manyToOneRel) {
+    fields.push({
+      name: 'manyToOneRel',
+      type: 'relation',
+      value: [{ label: manyToOneRel }],
+    } satisfies FieldValue);
+  }
+
+  if (manyToManyRel) {
+    fields.push({
+      name: 'manyToManyRel',
+      type: 'relation',
+      value: manyToManyRel.map((label) => ({ label })),
+    } satisfies FieldValue);
+  }
+
+  if (oneWayRel) {
+    fields.push({
+      name: 'oneWayRel',
+      type: 'relation',
+      value: [{ label: oneWayRel }],
+    } satisfies FieldValue);
+  }
+
+  if (selfRel) {
+    fields.push({
+      name: 'selfRel',
+      type: 'relation',
+      value: selfRel.map((label) => ({ label })),
+    } satisfies FieldValue);
+  }
+
+  return fields;
+};
+
+/**
+ * Base Relations Tests
+ *
+ * This file tests core relation functionality without Draft & Publish or i18n features.
+ * It focuses on the following operations:
+ * - Adding relations (single and multiple)
+ * - Removing relations
+ * - Reordering relations (for *-to-many relations)
+ * - CRUD operations for relations
+ */
+test.describe('Relations - EditView', () => {
+  test.beforeEach(async ({ page }) => {
+    await resetDatabaseAndImportDataFromPath('with-admin.tar');
+    await page.goto('/admin');
+    await login({ page });
+  });
+
+  test('Create a RelationSource entry with a one-to-many relation', async ({ page }) => {
+    const fields = createRelationSourceFields({ oneToManyRel: ['Target 1'] });
+    await createContent(page, 'Relation Source', fields, { save: true, verify: true });
+  });
```

Comment:
Does this test assert anything? There is another test file that asserts content creation: `create-content.spec.ts`. If we are trying to assert that the relation content is created correctly should it be done over there?